starter-code/src/functions-and-arrays.js

```diff
@@ -11,13 +20,31 @@ var words = [
   'crackpot'
 ];
 
+function findLongestWord (arrayLongWord) {
+  for (var i = 0; i < arrayLongWord.length; i++) {
+    if (longest < arrayLongWord[i].length) {
+      longest = arrayLongWord[1].length;
+      words=arrayLongWord[i];
+    }
+  }
+  return arrayLongWord;
+}
+
 // Calculating a Sum
 
-var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
+  var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
+  var total=0;
+  for(var i in numbers) { total += numbers[i]; }
 
 // Calculate the Average
 
 var numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
+var total=0;
+for(var i in numbersAvg) {total += numbersAvg [i]} {
+  var avg = total/numbersAvg.length;
+}
```

Comment:
The for in loop is recommended to be used with objects, here's what you could do:
```
function sumCalculation(arrayNumbers) {
  var total = 0;
  arrayNumbers.forEach(function (number) {
    total += number;
  });
  return total;
}
```